build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+cd $(dirname $0)
+src=$(pwd)
+cd - >/dev/null
+
+if [ ! -d build]; then
+    # configure a debug build (unoptimized and with debug info) for development
+    meson setup build --buildtype=debug
```

Comment:
It would probably be a good idea to use the canonical path here `/usr/bin/meson` to prevent collision (not that it should happen, but still best practice). Also, we should probably include a check that `Meson` is installed, as well.